src/theme/BlogPostItem/styles.module.css

```diff
@@ -19,7 +19,11 @@
 }
 .blogPostData {
   font-size: 0.9rem;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
```

Comment:
To center the 'Date + Reading time' when there is no link to the Forum, I'd use a centered layout and add `margin-left: auto;` to the 'Replies' component.
```suggestion
  justify-content: center;
```

Reply:
Closed on https://github.com/flashbots/flashbots-writings-website/pull/98